vite.config.ts

```diff
@@ -18,13 +18,18 @@ export default defineConfig(({ mode }) => {
 
   return {
     build: {
-      chunkSizeWarningLimit: 900,
       outDir,
       rollupOptions: {
         input: {
           main: resolve(__dirname, 'index.html'),
           _scrivito_extensions: resolve(__dirname, '_scrivito_extensions.html'),
         },
+        output: {
+          manualChunks: {
```

Comment:
Besides the newly created `react` and `scrivito` chunk there are still two js files in the end: `main-<hex>.js` and `index-<hex>.js` - since `main-<hex>.js` is quite small, can they be combined into one file?